src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -239,6 +232,12 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
+            if (token.IsKind(SyntaxKind.LessThanToken, SyntaxKind.GreaterThanToken) && token.Parent.IsKind(SyntaxKind.TypeParameterList, SyntaxKind.TypeArgumentList))
+            {
+                text = Keyword("generics");
```

Comment:
I'm using the `Keyword` call for consistency, but let me know if I shouldn't use it in this case.